src/geode/georepresentation/msh_input.cpp

```diff
@@ -132,38 +134,64 @@ namespace
             return elementary_entity_id_;
         }
 
-        const std::vector< geode::index_t >& element_vertices_id()
+    protected:
+        geode::index_t& nb_vertices()
         {
-            return element_vertices_id_;
+            return nb_vertices_;
+        }
+
+        std::vector< geode::index_t >& vertex_ids()
+        {
+            return vertex_ids_;
+        }
+
+        void read_vertex_ids()
+        {
+            std::string id_info;
+            vertex_ids().resize( nb_vertices() );
+            for( auto n : geode::Range{ nb_vertices() } )
+            {
+                iss_vertices_ids_ >> id_info;
+                vertex_ids()[n] = std::stoi( id_info );
+            }
         }
 
     private:
         geode::index_t physical_entity_id_;
         geode::index_t elementary_entity_id_;
-        std::vector< geode::index_t > element_vertices_id_;
+        std::istringstream& iss_vertices_ids_;
+        geode::index_t nb_vertices_{ 0 };
+        std::vector< geode::index_t > vertex_ids_;
     };
 
     using GMSHElementFactory = geode::Factory< geode::index_t,
         GMSHElement,
         geode::index_t,
         geode::index_t,
-        std::vector< geode::index_t > >;
+        std::istringstream& >;
 
     class GMSHPoint : public GMSHElement
     {
     public:
         GMSHPoint( geode::index_t physical_entity_id,
             geode::index_t elementary_entity_id,
-            std::vector< geode::index_t > element_vertices_id )
-            : GMSHElement( physical_entity_id,
-                  elementary_entity_id,
-                  element_vertices_id )
+            std::istringstream& iss_vertices_ids )
+            : GMSHElement(
+                  physical_entity_id, elementary_entity_id, iss_vertices_ids )
         {
+            nb_vertices() = 1;
             OPENGEODE_EXCEPTION( elementary_entity_id > 0,
                 "GMSH tag for elementary entity "
                 "(second tag) should not be null" );
-            OPENGEODE_EXCEPTION( element_vertices_id.size() == 1,
-                "GMSH element for point (type=15) should have one vertex" );
+            try
+            {
+                read_vertex_ids();
+            }
+            catch( ... )
+            {
+                geode::Logger::error(
+                    "GMSH element for point (type=15) should have one vertex" );
+            }
         }
         void add_element( geode::BRep& brep, GmshId2Uuids& id_map ) final
         {
```

Comment:
But { } works for cur_gmsh_id and builder

Reply:
I do not understand the point here.